server/app.ts

```diff
@@ -21,47 +18,52 @@ process.on("unhandledRejection", err => {
 	throw err;
 });
 
-interface IStatusReturn {
-	[medium: string]: [APIReturn];
+interface IPluginReturn {
+	plugin: string;
+	errors: APIReturn[];
 }
 
 const resolvers = {
 	Query: {
-		send_message: async (prev: any, args: any): Promise<IStatusReturn> => {
-			let statusRet: IStatusReturn = {};
+		send_message: async (prev: any, args: any): Promise<IPluginReturn[]> => {
 			const message = args.message;
 
-			const checkQueue = Object.keys(args.plugins).map( name => {
+			const checkQueue = Object.keys(args.plugins).map( rawName => {
 
 				return (async () => { // Loading checkQueue IIFE
-
-					const plugin: GenericNotifier<any> = plugins.mediaAPI[name.toUpperCase()];
-					const verifiedConfig = await plugin.check(args.plugins[name]); // Verify
+					// Upper Cameled
+					const name = rawName.split('_').map(w => w[0].toUpperCase() + w.slice(1)).join('');
```

Comment:
This makes me nervous about keeping this implementation in sync with the generation code, and making sure one is the reverse of the other. A few options here:
1. Write some tests to make sure that these conversions work (so they don't become out of sync in the future)
2. Don't make this conversion, for a plugin like `LiveSite` just have the type named `LiveSiteConfig` and the key for it be named `livesite` so you just have to do a `.toLowerCase()` and search through the map (or maybe make another map with the lowercased values)